src/serverOperations/crudOperations.js

```diff
@@ -0,0 +1,32 @@
+import { fetchFromBase } from './fetchFromBase';
+
+export async function createRecord(base, data, sessionID) {
+    return await fetchFromBase(`${base}/add`, { ...data, sessionID }, 'post');
+}
+
+export async function getRecords(
+    base,
+    fromRecordID,
+    recordAmount,
+    params,
+    sessionID,
+) {
+    fromRecordID = fromRecordID > 0 ? fromRecordID : undefined;
+    recordAmount = recordAmount > 0 ? recordAmount : undefined;
+
+    const query = { ...params, sessionID, fromRecordID, recordAmount };
+
+    return fetchFromBase(`${base}/get`, query);
```

Comment:
If you don't use `query` anywhere else - you don't usually need a dedicated variable for it. Same for other places in code.